core/src/main/resources/hudson/PluginManager/_table.js

```diff
@@ -504,24 +504,27 @@ window.addEventListener("load", function () {
     });
   });
 
-  // Enable/disable the 'Update' button depending on if any updates are checked
-  const anyCheckboxesSelected = () => {
-    return (
-      document.querySelectorAll("input[type='checkbox']:checked:not(:disabled)")
-        .length > 0
-    );
-  };
   const updateButton = document.querySelector("#button-update");
-  const checkboxes = document.querySelectorAll(
-    "input[type='checkbox'], [data-select], .jenkins-table__checkbox",
-  );
-  checkboxes.forEach((checkbox) => {
-    checkbox.addEventListener("click", () => {
-      setTimeout(() => {
-        updateButton.disabled = !anyCheckboxesSelected();
+  if (updateButton) {
```

Comment:
minor fix I noticed while checking this again, tests were logging a giant stacktrace, this button is only on the updates page and not the installed page so its event listener shouldn't be active